src/lib/api/api.ts

```diff
@@ -207,7 +207,8 @@ export default class Api extends EventEmitter {
       if (!result) throw new Error(`API ${ method } ${ endpoint } result undefined`)
       if (!this.success(result, ignore)) throw result
       this.logger && this.logger.debug(`[API] ${method} ${endpoint} result ${result.status}`)
-      return (method === 'DELETE') ? result : result.data
+      const hasDataInsideResult = result && !result.data;
```

Comment:
@MarcosSpessatto, just a suggestion, I would implement something like the statement below:
```
const { data } = result;
return (method === 'DELETE') && (data || result);
```

It's just a suggestion, your code is working fine as well.